src/fragmentarium/ui/SearchForms.test.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+import { Router } from 'react-router-dom'
+import { render } from '@testing-library/react'
+import { factory } from 'factory-girl'
+import Promise from 'bluebird'
+import SessionContext from 'auth/SessionContext'
+import { changeValueByLabel, whenClicked } from 'test-support/utils'
+import SearchForms from './SearchForms'
+import { createMemoryHistory } from 'history'
+import {
+  fillBibliographySelect,
+  expectedLabel,
+} from 'test-support/test-bibliographySelect'
+
+let number
+let id
+let title
+let primaryAuthor
+let year
+let pages
+let transliteration
+let fragmentService
+let fragmentSearchService
+let session
+let container
+let element
+let history
+
+let entry
+let searchEntry
```

Comment:
Use types.

Reply:
`
...
let fragmentService: any
let fragmentSearchService: any
let session: any
let element: RenderResult
let history: any
...
`
have types any the rest has proper types